Makefile

```diff
@@ -0,0 +1,23 @@
+FILES := main
+
+default: main
+
+.PHONY: all
+all:  deps main test
+
+.PHONY: deps
+deps:
```

Comment:
deps - is just shortcut for dependencies, right?
But there is a few more things we can do with dependencies: verify, download, 
When tidy is
> The go mod tidy command cleans up these unused dependencies

I don't have a good idea how to name it but maybe just `mod_tidy`, `tidy`